### DISCUSSION TITLE
Welcome to qualreas Discussions!

### DISCUSSION THREAD

**Discussion by alreich** [alreich/qualreas#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by erumharis-pk** [comment]
I am looking for some explanation on the use of ontology (Experimenting with Protege) in qualreas. There is no description on how to integrate or use the time.ttl ontology in qualreas or how it has been used if already? Any help would be highly appreciated.

**Reply by alreich** [reply]
That is a good question. The short answer is that, currently, _qualreas_ is **not** integrated with OWL/RDF.

The W3C ontology of time (http://www.w3.org/2006/time) includes Allen's 13 relations between proper time intervals. The reason I included the file, "timeX.ttl", was simply to show how that can be extended to include time points (instants) as well (See https://qualreas.readthedocs.io/en/latest/extended_interval_algebras.html#linear-interval-and-point-algebra)

Now, what I'm about to say might be outdated or even just incorrect, but to **MY** knowledge, not only is there no standardized way to express a temporal constraint network in OWL/RDF format, using only Allen's relations, but also none of the current OWL Reasoners (HermiT, Racer, etc) support the type of constraint processing required by such a temporal constraint network. For a very simple example of a temporal constraint network and temporal reasoning, see https://qualreas.readthedocs.io/en/latest/temporal_reasoning_about_intervals_and_points.html#an-example-of-temporal-reasoning

I would love to be proven wrong about what I just said. If so, I would consider integrating qualreas with OWL/RDF.

Thank you for the question.
